codersdk/workspaceapps.go

```diff
@@ -4,12 +4,30 @@ import (
 	"github.com/google/uuid"
 )
 
+type WorkspaceAppHealth string
+
+const (
+	WorkspaceAppHealthDisabled     WorkspaceAppHealth = "disabled"
+	WorkspaceAppHealthInitializing WorkspaceAppHealth = "initializing"
+	WorkspaceAppHealthHealthy      WorkspaceAppHealth = "healthy"
+	WorkspaceAppHealthUnhealthy    WorkspaceAppHealth = "unhealthy"
+)
+
 type WorkspaceApp struct {
 	ID uuid.UUID `json:"id"`
 	// Name is a unique identifier attached to an agent.
 	Name    string `json:"name"`
 	Command string `json:"command,omitempty"`
 	// Icon is a relative path or external URL that specifies
 	// an icon to be displayed in the dashboard.
-	Icon string `json:"icon,omitempty"`
+	Icon                 string             `json:"icon,omitempty"`
+	HealthcheckEnabled   bool               `json:"healthcheck_enabled"`
+	HealthcheckPeriod    int32              `json:"healthcheck_period"`
+	HealthcheckThreshold int32              `json:"healthcheck_threshold"`
+	Health               WorkspaceAppHealth `json:"health"`
```

Comment:
I might call this `HealthEnabled` instead to align with `Health`.

Reply:
I left these as all "healthcheck" because these are the readonly values that will come from the template, while health is the dynamic computed value.